Earthfile

```diff
@@ -1,11 +1,46 @@
 VERSION 0.6
 FROM gradle:7.1.0-jdk11
+RUN apt-get update && apt-get install -y \
+  zip \
+  && rm -rf /var/lib/apt/lists/*
+ARG version=0.0.0
+ARG bundle
 COPY settings.gradle.kts build.gradle.kts ./
+COPY scripts scripts
 COPY src src
 
-test:
-  RUN --mount=type=cache,target=/root/.gradle/caches gradle test --tests LexerTests
+GET_BUNDLE:
+  COMMAND
+  COPY github.com/earthly/earthly/contrib/earthfile-syntax-highlighting+export/ build/
+  # COPY ../earthly/contrib/earthfile-syntax-highlighting+export/ build/
```

Comment:
You could make this an ARG if you need it for local dev. See for example: https://github.com/earthly/earthly/blob/main/Earthfile#L44